tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_8_tensorrt8_6

```diff
@@ -41,6 +41,8 @@ RUN v="8.6.1.6-1+cuda11.8" &&\
 # Compile trtexec
 RUN cd /usr/src/tensorrt/samples/trtexec && make
 
+# Install Valgrind
```

Comment:
How about the other docker files? Do you also need to update them?

Reply:
Yes, I will add another commit to update them as well. Thanks for pointing that out!